packages/flutter/test/material/bottom_sheet_theme_test.dart

```diff
@@ -132,6 +132,34 @@ void main() {
     expect(material.elevation, elevation);
     expect(material.shape, shape);
   });
+
+  testWidgets('BottomSheet persistentElevation takes priority over elevation', (WidgetTester tester) async {
```

Comment:
I think more test cases are needed, since there a are a few different combinations:
show a persistent bottom sheet vs bottom sheet with different combinations of persistentElevation and elevation being defined

Reply:
Done